js/components/dataset/list.vue

```diff
@@ -21,7 +21,8 @@ export default {
                 key: 'title',
                 sort: 'title',
                 align: 'left',
-                type: 'text'
+                type: 'text',
+                width: 200
```

Comment:
I think we should bring back an older solution we had: having a grouped header `Update` with 2 nested columns: `Data` and `Metadata`.